pkg/coremodel/playlist/coremodel.cue

```diff
@@ -0,0 +1,161 @@
+package playlist
+
+import (
+	"strings"
+	"github.com/grafana/thema"
+)
+
+let PlaceholderPrefix = "placeholder-cupcake-"
+
+thema.#Lineage
+name: "playlist"
+seqs: [
+	{
+		schemas: [
+			{//0.0 - represents existing Grafana playlist type
+				// Serial database identifier of the playlist.
+				id: int64
+				// Unique playlist identifier. Generated on creation, either by the
+				// creator of the playlist of by the application.
+				uid: string
+				// Name of the playlist.
+				name: string
+				// Interval sets the time between switching views in a playlist.
+				// FIXME: Is this based on a standardized format or what options are available? Can datemath be used?
+				interval: string | *"5m"
+				// The ordered list of items that the playlist will iterate over.
+				items?: [...#Item]
+
+				///////////////////////////////////////
+				// Definitions (referenced above) are declared below
+
+				#Type: "dashboard_by_id" | "dashboard_by_tag" | "dashboard_by_uid" @cuetsy(kind="enum",memberNames="DashboardById|DashboardByTag|DashboardByUID")
```

Comment:
Any thoughts on just keeping `dashboard_by_id` and marking it deprecated?

Reply:
Its deprecation is marked in a comment below, but yeah it should be marked as deprecated here, as well.

I think that the lens we wrote to handle removing it in the `1.0` does a good job of preserving an id value, so there's no downside to just pulling the trigger and getting rid of that possibility.

Reply:
I really meant "just" ie -- skipping all the migration stuff

Reply:
yea, the migration looks to add more complexity than it solves